### ISSUE TITLE
Restoring session puts all tabs in a single group

### ISSUE THREAD

**Issue by Warix3** [Rapptz/power-tabs#44]
When my PC loses power and if firefox was working then after I turn on PC again and turn on firefox there is a screen saying firefox was shut down improperly and if I use the option to restore the session then all of the tabs will be restored but put in a single last used group. 
Firefox version: 67.0.2 (64-bit)
OS version: Windows 10 Pro 1809

**Comment by stefan-vogel-stalder** [comment]
Same problem under Linux/openSuSE Leap 15.1 FF: 60.8.0esr (64-bit)
After updating to FF 68.1 the problem went away

**Comment by KunjanChauhan** [comment]
Yes, I have this issue as well.  There are times when FF crashes and upon restart all tab groupings are lost with tabs being reassigned to the pre-crash active tab. Is it possible to either build in a session backup function which stores a table of tabs and their groupings and the date time of the snapshot with an ability to restore or to integrate e.g. the "Tab Session Manager" or similar plugin? ty.
